spec/models/spree/variant_spec.rb

```diff
@@ -1,3 +1,4 @@
 RSpec.describe Spree::Variant, type: :model do
   it { is_expected.to have_many(:volume_prices) }
+  it { is_expected.to have_many(:model_volume_prices) }
```

Comment:
I absolutely agree. Was just repeating the style of testing from above. Will leave it, though :)